swagger.yaml

```diff
@@ -2974,6 +2974,42 @@ paths:
           description: Error.
           schema:
             $ref: '#/definitions/error'
+    post:
+      tags:
+        - bootfiles
+      security:
+        - userAuth: [admin, read-only-admin, user]
+      description: Uploads boot-files according to the specified openshift-version.
+      operationId: UploadBootFiles
```

Comment:
This name is actually confusing. Because you are now capable of changing all the `openshift-version` fields (should it be possible?) including `support-level` for example. This isn't related to files.

Reply:
right, removed for now.